ui/src/dashboards/components/dashboard_index/Table.tsx

```diff
@@ -124,6 +124,14 @@ class DashboardsTable extends PureComponent<Props, State> {
     this.setState({sortKey, sortDirection: nextSort, sortType})
   }
 
+  private summonImportFromTemplateOverlay = (): void => {
+    const {
+      router,
+      params: {orgID},
+    } = this.props
+    router.push(`/orgs/${orgID}/dashboards/templates`)
```

Comment:
this route url should have something in it to indicate that its import `/orgs/${orgID}/dashboards/import/template` would be my go to.